src/Cake.Npm/Install/NpmInstallSettingsExtensions.cs

```diff
@@ -106,6 +106,53 @@ public static NpmInstallSettings ForProduction(this NpmInstallSettings settings)
         }
 
         /// <summary>
+        /// Defines that npm should npm should prevent optional dependencies from being installed
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="NpmInstallSettings.NoOptional"/> set to <c>true</c>.</returns>
+        public static NpmInstallSettings WithNoOptional(this NpmInstallSettings settings)
+        {
+            if (settings == null)
+            {
+                throw new ArgumentNullException(nameof(settings));
+            }
+
+            return settings.WithNoOptional(true);
+        }
+
+        /// <summary>
+        /// Defines that npm should not prevent optional dependencies from being installed
```

Comment:
Comment is wrong. If `--no-optional` is not provided optional dependencies will be installed.